website/versioned_docs/version-0.8.0/flink-quick-start-guide.md

```diff
@@ -219,18 +220,31 @@ allocate enough memory, we can try to set these memory options.
 
 |  Option Name  | Description | Default | Remarks |
 |  -----------  | -------  | ------- | ------- |
-| `write.tasks` |  The parallelism of writer operators. If value is `N`, each writer writes 1 to `N` buckets in sequence. Default `4` | `4` | Increasing the parallelism has no effect on the number of small files |
-| `write.bucket_assign.tasks`  |  The parallelism of bucket assigner operators. No default value, using flink `parallelism.default`  | [`parallelism.default`](#parallelism) |  Increasing the parallelism also increases the number of buckets, which increases the number of small files (small buckets) in disguise.  |
-| `write.index_boostrap.tasks` |  The parallelism of index bootstrap. Increasing parallelism can speed up the efficiency of the bootstrap stage. The bootstrap stage will block checkpointing. Therefore, it is necessary to set more checkpoint failure tolerance times. No default value, using Flink `parallelism.default` | [`parallelism.default`](#parallelism) | It only take effect when `index.bootsrap.enabled` is `true` |
+| `write.tasks` |  The parallelism of writer tasks. Each write task writes 1 to `N` buckets in sequence. Default `4` | `4` | Increases the parallelism has no effect on the number of small files |
+| `write.bucket_assign.tasks`  |  The parallelism of bucket assigner operators. No default value, using flink `parallelism.default`  | [`parallelism.default`](#parallelism) |  Increases the parallelism also increases the number of buckets, thus the number of small files (small buckets)  |
+| `write.index_boostrap.tasks` |  The parallelism of index bootstrap. Increasing parallelism can speed up the efficiency of the bootstrap stage. The bootstrap stage will block checkpointing. Therefore, it is necessary to set more checkpoint failure tolerance times. Default using Flink `parallelism.default` | [`parallelism.default`](#parallelism) | It only take effect when `index.bootsrap.enabled` is `true` |
 | `read.tasks` | The parallelism of read operators (batch and stream). Default `4`  | `4` |  |
 | `compaction.tasks` | The parallelism of online compaction. Default `10` | `10` | `Online compaction` will occupy the resources of the write task. It is recommended to use [`offline compaction`](#offline-compaction) |
 
-### Others
+### Compaction
+
+:::note
+These are options only for `online compaction`. 
+:::
+
+:::note
+Turn off online compaction by setting `compaction.async.enabled` = `false`, but we still recommend turning on `compaction.schedule.enable` during writing, and then directly execute the compaction plan generated during writing by [`offline compaction`](#offline-compaction). 
+:::
```

Comment:
during writing -> For the writing job

and then directly execute the compaction plan generated during writing -> you can then execute the compaction plan